homeassistant/components/media_player/plex.py

```diff
@@ -38,6 +39,8 @@
 CONF_USE_EPISODE_ART = 'use_episode_art'
 CONF_USE_CUSTOM_ENTITY_IDS = 'use_custom_entity_ids'
 CONF_SHOW_ALL_CONTROLS = 'show_all_controls'
+CONF_PURGE_UNAVAILABLE_CLIENTS = 'purge_unavailable_clients'
+CONF_CLIENT_PURGE_INTERVAL = 'client_purge_interval'
```

Comment:
1. Do we call them "clients" anywhere else? I think "player" is a better term. Client is code, player is user-facing. :) But if there's already precedent here then just do what's consistent.
2. I would remove the word purge here. Purge sounds like a retroactive destruction, removing all history of a thing. I suggest a softer word like "remove"